src/spatial/views/chatbot_views.py

```
@@ -0,0 +1,90 @@
+# views/chatbot_views.py
+from flask import request, jsonify
+from models.chatbot_model import AirQualityChatbot, DataFetcher  
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)